### ISSUE TITLE
Do you plan to support Windows platform?

### ISSUE THREAD

**Issue by solvingj** [codrsquad/portable-python#4]
Are you planning to support windows in these projects (`portable-python` and `runez`, etc)?  If so, I suggest adding windows builds to your CI.  If not, it's sad because `portable-python` project is so perfect in terms of workflow. 

I ask because I just opened this issue and I'm not sure that this project could have ever run successfully on Windows with this condition existing: https://github.com/codrsquad/runez/issues/5

**Comment by zsimic** [comment]
I'd love to get around to making it work on windows, it's not a priority for me right now because I don't have windows use-cases yet.
I did a little bit of exploring, fixing a few things in `runez` on Windows a long time ago.
I agree, would be super cool to make it all work on Windows too.

**Comment by solvingj** [comment]
Ok, thank you for the fast feedback.  Perhaps consider adding a statement to the readme that this doesn't currently support Windows, because I got pretty far along in a POC before hitting this and it was super surprising and disappointing.  It's on me for making assumptions because I got used to most python CLI apps working across all three major platforms by default. So, no hard feelings, just a shout out to help future consumers.
